assets/scripts/main.js

```diff
@@ -3,47 +3,32 @@
  *
  */
 
-// Initalization scripts
-import './app/initialization'
+import $ from 'jquery'
+window.$ = $
 
-// Export modules to window for the global-scoped JavaScript
-import './exports'
+import _ from 'lodash'
+window._ = _
 
-// Load things that are needed
-import './app/event_tracking'
+import { Stmx } from './app/initialization'
+window.Stmx = Stmx
+
+// import modules for side-effects
 import './app/blocking_shield'
 import './app/debug_info'
 import './app/keyboard_commands'
 import './app/print'
 import './app/status_message'
 import './app/welcome'
-
-// Menus
-import './menus/menu'
-import './menus/_feedback'
-import './menus/_help'
-import './menus/_identity'
-import './menus/_settings'
-import './menus/_share'
-
-// Dialogs
-import './dialogs/_about'
-import './dialogs/_save_as_image'
```

Comment:
I see, the `saveAs`. The word polyfill is unfortunate here, as the `FileSaver` API has been discontinued and is no longer part of the HTML5 spec. So it's not so much a polyfill anymore as it is merely a cross-browser file saving implementation. If were a true polyfill, I'd be happy to move it into `main.js`. But since we only use it here, it might make more sense to leave it.